website/docs/r/orbital_contact_profile.html.markdown

```diff
@@ -0,0 +1,166 @@
+
+* `resource_group_name` - (Required) The name of the Resource Group where the contact profile exists.
+
+* `location` - (Required) The location where the contact profile exists.
+
+* `minimum_variable_contact_duration` - (Required) Minimum viable contact duration in ISO 8601 format. Used for listing the available contacts with a spacecraft at a given ground station.
+
+* `auto_tracking` - (Required) Auto-tracking configurations for a spacecraft. Possible values are `disabled`, `xBand` and `sBand`.
+
+* `network_configuration_subnet_id` - (Required) ARM resource identifier of the subnet delegated to the Microsoft.Orbital/orbitalGateways. Needs to be at least a class C subnet, and should not have any IP created in it.
+
+* `links` - (Required) A list of spacecraft links. A `links` block as defined below.
+
+* `event_hub_uri` - (Optional) ARM resource identifier of the Event Hub used for telemetry. Requires granting Orbital Resource Provider the rights to send telemetry into the hub.
+
+* `minimum_elevation_degrees` - (Optional) Maximum elevation of the antenna during the contact in decimal degrees.
+
+---
+
+A `links` block supports the following:
+
+* `channels` - (Required) A list of contact profile link channels. A `channel` block as defined below.
+
+* `direction` - (Required) Direction of the link. Possible values are `Uplink` and `Downlink`.
+
+* `name` - (Required) Name of the link.
+
+* `polarization` - (Required) Polarization of the link. Possible values are `RHCP`, `LHCP`, `linearVertical` and `linearHorizonal`.
+
+---
+
+A `channels` block supports the following:
+
+* `name` - (Required) Name of the channel.
+
+* `center_frequency_mhz` - (Required) Center frequency in MHz.
+
+* `bandwidth_mhz` - (Required) Bandwidth in MHz.
+
+* `end_point` - (Required) Customer End point to store/retrieve data during a contact. An `end_point` block as defined below.
+
+* `modulation_configuration` - (Optional) Copy of the modem configuration file such as Kratos QRadio. Only valid for uplink directions. If provided, the modem connects to the customer endpoint and accepts commands from the customer instead of a VITA.49 stream.
+
+* `demodulation_configuration` - (Optional) Copy of the modem configuration file such as Kratos QRadio or Kratos QuantumRx. Only valid for downlink directions. If provided, the modem connects to the customer endpoint and sends demodulated data instead of a VITA.49 stream.
+
+* `encoding_configuration` - (Optional) Configuration for encoding the channel.
+
+* `decoding_configuration` - (Optional) Configuration for decoding the channel.
+
+---
+
+An `end_point` block supports the following:
+
+* `end_point_name` -(Required) Name of an end point.
+
+* `ip_address` - (Required) IP address of an end point.
+
+* `port` - (Required) TCP port to listen on to receive data.
+
+* `protocol` - (Required) Protocol of an end point. Possible values are `TCP` and `UDP`.
+
```

Comment:
Not all blocks or properties are documented here

Reply:
Removed unused properties.